sections/04_study.md

```diff
@@ -241,114 +241,61 @@ this was easy to simulate with a computational model. As we learn to better
-translation are still not well understood, compounded by incomplete
-understanding of alternative transcripts and the context for these alternatives.
-Sequence similarity is poor even between functionally correlated genes. While
-homologs might be studied for insight, they may not exist or may be just as
-poorly characterized.
-
-Recognizing enhancers presents additional challenges. Enhancers may be up to one
-million base pairs upstream or downstream from the affected promoter on either
-strand and even within the introns of other genes [@doi:10.1038/nrg3458]. They
-do not necessarily operate on the nearest gene and may affect multiple genes.
-Their activity is frequently tissue- or context-specific. Many
-enhancers display modest or no conservation across species. There is no
-universal enhancer sequence signal or marker for enhancers, and some literature
-suggests that enhancers and promoters may be just categories along a spectrum
-[@doi:10.1016/j.tig.2015.05.007]. One study [@doi:10.1101/gr.173518.114] even
-showed that only 33% of predicted regulatory regions could be validated, while a
-class of "weak" predicted enhancers were strong drivers of expression. Yet there
-is growing evidence for their vast ubiquity, making them possibly the
-predominant functional non-coding element. Thus, identifying enhancers is
-critical yet the search space is large.
-
-While previous (non-deep learning) approaches have steadily improved
-promoter prediction, there is little consensus on the best approach and
-performance is poor. Typically algorithms will recognize only half of all
-promoters, with an accompanying high false positive rate
-[@doi:10.1101/gr.7.9.861]. Methods with better sensitivity generally do so at
-the cost of poorer specificity. Conventional identification of enhancers has
-leaned heavily on simple conservation or laborious experimental techniques, with
-only moderate sensitivity and specificity. For example, while chromatin
-accessibility has often been used for identifying enhancers, this also
-"recognizes" a wide variety of other functional elements, like promoters,
-silencers, and repressors.
-
-The complex nature of CREs and our lack of understanding makes them a natural
-candidate for deep learning approaches. Indeed, neural networks were used for
-promoter recognition as early as 1996, albeit with mixed results
-[@doi:10.1016/S0097-8485(96)80015-5]. Since then, there has been much work in
-applying deep learning to this area, although little in the way of comparative
-studies or formal benchmarks. We therefore focus on a few recent important
-studies to outline the state of the art and extant problems.
-
-Basset [@doi:10.1101/gr.200535.115] trained CNNs on DNA accessibility datasets,
-getting a marked improvement on previous methods, albeit still with a high false
-positive rate. The multi-task architecture resembles DeepSEA
-[@tag:Zhou2015_deep_sea], which predicted open chromatin regions and histone
```

Comment:
DeepSEA is still cited in the previous subsection which seems like a more appropriate place. It does not seem to have anything to do with promoters or enhancers per se; the words "promoter" and "enhancer" do not even appear in its text.